src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -203,31 +210,48 @@ export const App = ({ displayToggle, isLOA1, loggedIn, toggleLoginModal }) => {
     </VaAlertSignIn>
   );
 
-  if (loggedIn) {
-    if (!displayToggle) {
-      return unavailableComponent();
-    }
-    if (formError.type === errorTypes.NOT_FOUND) {
-      return notFoundComponent();
+  const getVerifiedComponent = () => {
+    if (formError.type === errorTypes.SYSTEM_ERROR) {
+      return systemErrorComponent;
     }
-    if (isLOA1) {
-      return verifyAlertVariant;
+    if (profile.isLoggedIn) {
+      if (!displayToggle) {
+        return unavailableComponent();
+      }
+      if (formError.type === errorTypes.NOT_FOUND) {
+        return notFoundComponent();
+      }
+      return downloadForm;
     }
-    return downloadForm;
+    return null;
```

Comment:
Is this null condition possible? I'd guess not because this method is only called if user is loa3, and I'd speculate it's not possible for the user to loa3 without being logged in. I wonder if you can just remove the `profile.isLoggedIn` condition.

Reply:
Good catch! Removed.